libs/auth/src/openssl.test.ts

```diff
@@ -12,8 +11,14 @@ import {
 import { createCert, openssl } from './openssl';
 
 jest.mock('child_process');
-jest.mock('fs');
-jest.mock('fs/promises');
+jest.mock('fs', () => ({
+  existsSync: jest.fn(),
+  promises: {
+    mkdir: jest.fn(),
+    unlink: jest.fn(),
+    writeFile: jest.fn(),
+  },
+}));
```

Comment:
Why this change? Just curious

Reply:
This is just a side effect of having to change how we import `fs/promises`. `jest.mock(...)` only mocks one level deep in a library.

So while this mocks all the `fs` promise methods:

```ts
import * as fs from 'fs/promises';

jest.mock('fs/promises');
```

This does not:

```ts
import { promises as fs } from 'fs';

jest.mock('fs');
```